Content.Client/UserInterface/Screens/SeparatedChatGameScreen.xaml.cs

```diff
@@ -1,14 +1,25 @@
 using System.Numerics;
 using Content.Client.UserInterface.Systems.Chat.Widgets;
+using Content.Shared.CCVar;
 using Robust.Client.AutoGenerated;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface;
 using Robust.Client.UserInterface.Controls;
 using Robust.Client.UserInterface.XAML;
+using Robust.Shared;
+using Robust.Shared.Configuration;
 
 namespace Content.Client.UserInterface.Screens;
 
 [GenerateTypedNameReferences]
 public sealed partial class SeparatedChatGameScreen : InGameScreen
 {
+    private readonly IConfigurationManager _cfg = IoCManager.Resolve<IConfigurationManager>();
+    [Dependency] private readonly IUserInterfaceManager _uiManager = default!;
+
+    private readonly float _chatMinWidth;
+    private float? _deferredSplitFraction = null;
```

Comment:
Nullable values default to null, so you don't need to explicitly set it.

```suggestion
    private float? _deferredSplitFraction;
```